lti_consumer/models.py

```diff
@@ -150,6 +150,34 @@ class LtiConfiguration(models.Model):
         help_text=_("Client ID used by LTI tool"),
     )
 
+    lti_1p3_oidc_url = models.CharField(
+        max_length=255,
+        blank=True,
+        help_text='This is the OIDC third-party initiated login endpoint URL in the LTI 1.3 flow, '
+                  'which should be provided by the LTI Tool.'
+    )
+
+    lti_1p3_launch_url = models.CharField(
+        max_length=255,
+        blank=True,
+        help_text='This is the LTI launch URL, otherwise known as the target_link_uri.'
+                  'It represents the LTI resource to launch to or load in the second leg of the launch flow, '
+                  'when the resource is actually launched or loaded.'
+    )
+
+    lti_1p3_tool_public_key = models.TextField(
+        blank=True,
+        help_text='This is the Tool\'s public key. This should be provided by the LTI Tool. '
+                  'One of either lti_1p3_tool_public_key or lti_1p3_tool_keyset_url must not be blank.'
+    )
+
+    lti_1p3_tool_keyset_url = models.CharField(
+        max_length=255,
+        blank=True,
+        help_text='This is the Tool\'s JWK (JSON Web Key) Keyset (JWKS) URL. This should be provided by the LTI Tool.'
+                  ' One of either lti_1p3_tool_public_key or lti_1p3_tool_keyset_url must not be blank.'
+    )
+
```

Comment:
Note: I did not add the `lti_1p3_tool_key_mode`, which is a field on the xBlock, to the model. Its only use is to toggle which field is displayed in the Studio view and to validate that `lti_1p3_tool_keyset_url` and `lti_1p3_tool_public_key` have valid values  and it doesn't seem to be used otherwise. I didn't feel it was necessary to include this in the model, since I can raise an exception in the `clean` method.

Reply:
makes sense to me 👍